histomicstk/features/ComputeIntensityFeatures.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+import pandas as pd
+import scipy.stats
+from skimage.measure import regionprops
+
+
+def ComputeIntensityFeatures(im_label, im_intensity,
+                             num_hist_bins=10, rprops=None):
+    """
+    Calculates intensity features from an intensity image.
+
+    Parameters
+    ----------
+    im_label : array_like
+        A labeled mask image wherein intensity of a pixel is the ID of the
+        object it belongs to. Non-zero values are considered to be foreground
+        objects.
+
+    im_intensity : array_like
+        Intensity image
+
+    num_hist_bins: int
+        Number of bins used to computed the intensity histogram of an object.
+        Histogram is used to energy and entropy features.
+
+    rprops : output of skimage.measure.regionprops, optional
+        rprops = skimage.measure.regionprops( im_label ). If rprops is not
+        passed then it will be computed inside which will increase the
+        computation time.
```

Comment:
@slee172 @cooperlab Optionally passing in output of regionprops to speed up computational time.